### PR TITLE
[wallet] Switch over to new tab bar design

### PR DESCRIPTION
**Pull request by cmcewen**
### Description

This swaps over to a new tab bar design to make the Send button stand out a little more. It also make the Send flow a stack on top of the tab bar again.

### Tested
Screenshot_20190906-183735
Screenshot_20190906-183739

### CODE REVIEW

**Review by martinvol — APPROVED** [review]

File: packages/mobile/src/navigator/TabNavigator.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const mapStateToProps = (state: RootState): StateProps => ({
 class WalletIconWithCircle extends React.Component<Props> {
   render() {
     return (
-      <View>
+      <View style={{ marginLeft: 3 }}>
```
Comment (jmrossy) [inline packages/mobile/src/navigator/TabNavigator.tsx]: 🚨🚨🚨

File: packages/mobile/src/navigator/TabNavigator.tsx [hunk 2]
```diff
@@ -95,22 +97,23 @@ export const TabNavigator = createBottomTabNavigator(
...
     [Screens.Send]: {
-      screen: Send,
+      screen: SendStack,
       navigationOptions: {
         tabBarButtonComponent: TabBarButtonComponent,
         tabBarIcon: (props: any) => <PaymentsIcon color={props.tintColor} />,
-        tabBarLabel: ({ tintColor }: LabelProps) => {
-          return <MenuText testID="SendNavigator" transKey="payments" tintColor={tintColor} />
+        tabBarLabel: () => null,
+        tabBarOnPress: () => {
```
Comment (jmrossy) [inline packages/mobile/src/navigator/TabNavigator.tsx:105]: Why is this necessary? Doesn't it do this by default based on the `screen` property above

File: packages/mobile/src/navigator/TabNavigator.tsx (hunk 1 above)
Comment (cmcewen) [reply packages/mobile/src/navigator/TabNavigator.tsx]: Thanks - forgot to push commit fixing this

File: packages/mobile/src/navigator/TabNavigator.tsx (hunk 2 above)
Comment (cmcewen) [reply packages/mobile/src/navigator/TabNavigator.tsx:105]: We wanted to switch back to the "modal" view, so it should pop on the stack again and not actually switch tabs
